modular_skyrat/modules/clock_cult/code/items/clockwork_slab.dm

```diff
@@ -173,6 +173,7 @@ GLOBAL_LIST_INIT(clockwork_slabs, list())
 			"purchased" = (scripture.type in purchased_scriptures),
 			"cog_cost" = scripture.cogs_required,
 			"typepath" = scripture.type,
+			"research_required" = !!(scripture.research_required ? !(scripture.type in GLOB.clockwork_research_unlocked_scriptures) : FALSE),
```

Comment:
I don't think I've seen two !'s used together, what does that do?

Reply:
converts a value to a boolean

Reply:
makes sense.